core/src/main/kotlin/com/tschuchort/compiletesting/KotlinCompilation.kt

```diff
@@ -71,6 +71,7 @@ class KotlinCompilation : AbstractKotlinCompilation<K2JVMCompilerArguments>() {
 	var javaParameters: Boolean = false
 
 	/** Use the IR backend */
+	@Deprecated("Since Kotlin 1.9.20 this option is no longer supported by the compiler. It has no effect.")
```

Comment:
Removed in [this commit](https://github.com/JetBrains/kotlin/commit/60016d3e5bb7c58d54bff3c082edfd53d0086218).